docs/content/getting-started/releases.mdx

```diff
@@ -1,28 +1,41 @@
 ---
-title: Releases and Changes | Dagster
+title: Releases and Compatibility | Dagster
 ---
 
-# Releases and Changes
+# Releases and Compability
 
-## Releases
+We follow [Semantic Versioning](https://semver.org/) for compatibility between Dagster releases.
 
-We currently publish two kinds of Dagster releases:
+## Dagster core
```

Comment:
```suggestion
---

## Dagster core
```
[1]
Adding a horizontal rule to visually separate sections. I've started doing this in other places, and it makes a big difference in lengthier content. Noting it here for consistency.